my_game.py

```diff
@@ -207,6 +235,13 @@ def on_update(self, delta_time):
 
         # Update the player shots
         self.player_shot_list.update()
+        
+        #Update Asteroids
+        self.asteroid_list.update()
+
+        #Temporary Asteroid test
+        new_asteroid = Asteroid()
+        self.asteroid_list.append(new_asteroid)
```

Comment:
Looks to me like this will add an Asteroid on on every frame. Please add the single Asteroid in the setup() function.

Reply:
Jeg kan se hvad du mener, jeg får gjort noget ved det.

Reply:
Done